src/datasets/once/once_scene_iterator.py

```diff
@@ -39,26 +46,19 @@ def __next__(self) -> tuple[str, FrameDescriptor]:
 
         frame_info = {}
         instance_ids = []
-
         frame_id = self.__current_sample
-        anno_json = get_annotations_tracked_file_name(self.__once.data_folder, self.__scene)
-        with open(anno_json, 'r') as json_file:
-            data = json.load(json_file)
-
-        while self.__current_id < len(self.__frame_ids):
-            print("Iterating frame " + str(self.__current_id + 1) + " of " + str(len(self.__frame_ids)))
-            if 'frames' in data and frame_id in {frame.get('frame_id') for frame in data['frames']}:
-                frame_info = next(frame for frame in data['frames'] if frame.get('frame_id') == frame_id)
-                if 'annos' in frame_info:
-                    instance_ids = frame_info['annos']['instance_ids']
-                    self.__current_id += 1
-                    if self.__current_id < len(self.__frame_ids):
-                        self.__current_sample = self.__frame_ids[self.__current_id]
-                    break  # found annotations
-            self.__current_id += 1
-            if self.__current_id < len(self.__frame_ids):
-                self.__current_sample = self.__frame_ids[self.__current_id]
-                frame_id = self.__current_sample
+
+
+        print("Iterating frame " + str(self.__current_id + 1) + " of " + str(len(self.__frame_ids)))

```

Comment:
let's remove prints as we have logging in `patch_scene.py`

Reply:
removed